.github/workflows/ci.yml

```diff
@@ -30,12 +30,17 @@ jobs:
       matrix:
         # Test latest and oldest supported Python releases
         # See https://devguide.python.org/versions/
-        python-version: ['3.9', '3.12']
+        python-version: ['3.11', '3.12']

```

Comment:
I'd prefer we test for 3.9 while we still support 3.9 to avoid any subtle errors creeping up and bricking our users' installations for those that are on 3.9